### PR TITLE
Feedback fixes

### PR DESCRIPTION
**Pull request by kikiya**
Commits on this PR address missing solution details and exercise setup changes to no longer assume persisted state between exercises.

### CODE REVIEW

File: exercises/load_exercise.bat [hunk 1]
```diff
@@ -28,16 +28,16 @@ if [!EXERCISE_FOLDER!]==[] (
 )
 
 for %%f in (%SUB_FOLDERS%) do (
-    set SOLUTION=!EXERCISE_FOLDER!\%%f
+    set SETUP=!EXERCISE_FOLDER!\%%f
     set EXERCISE=%%f
 
-    echo Loading Tests from !SOLUTION! to !EXERCISE!
+    echo Loading Tests from !SETUP! to !EXERCISE!
```
Comment (alexyarosh) [inline exercises/load_exercise.bat:34]:
```suggestion
    echo Loading setup files from !SETUP! to !EXERCISE!
```

File: exercises/load_exercise.sh [hunk 2]
```diff
@@ -22,32 +22,36 @@ fi
...
 fi
 
 for folder in ${SUB_FOLDERS[@]};
 do
-    SOLUTION=$EXERCISE_FOLDER/$folder
+    SETUP=$EXERCISE_FOLDER/$folder
     EXERCISE=./$folder
     
-    echo "Loading Tests from $SOLUTION to $EXERCISE"
+    echo "Loading Tests from $SETUP to $EXERCISE"
```
Comment (alexyarosh) [inline exercises/load_exercise.sh:47]:
```suggestion
    echo "Loading setup files from $SETUP to $EXERCISE"
```

File: setup/05-global-tables/movr/pricing/data/pricing_database.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+CREATE DATABASE movr_pricing;
```
Comment (alexyarosh) [inline setup/05-global-tables/movr/pricing/data/pricing_database.sql:1]: init_db.sh doesn't run this file, so the pricing db is not created

File: setup/06-regional-by-row/movr/users/data/users_database.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE movr_users;
+
+CREATE TABLE movr_users.users (
+	email STRING PRIMARY KEY,
+	city STRING NULL,
+	last_name STRING NOT NULL,
+	first_name STRING NOT NULL,
+	phone_numbers STRING[] NOT NULL
+);
```
Comment (alexyarosh) [inline setup/06-regional-by-row/movr/users/data/users_database.sql:9]:
```suggestion
);

ALTER DATABASE movr_users SET PRIMARY REGION "us-east";
ALTER DATABASE movr_users ADD REGION "us-central";
ALTER DATABASE movr_users ADD REGION "us-west";
```

The question in the instructions assumes that the regions have been added

File: setup/07-regional-tables/movr/pricing/data/pricing_database.sql [hunk 5]
```diff
@@ -0,0 +1,15 @@
+CREATE DATABASE movr_pricing;
```
Comment (alexyarosh) [inline setup/07-regional-tables/movr/pricing/data/pricing_database.sql:1]: init_db doesn't run this code

File: solutions/07-regional-tables/movr/pricing/data/pricing_database.sql [hunk 6]
```diff
@@ -16,7 +16,7 @@ CREATE TABLE movr_pricing.vip_rates (
     rate_code STRING PRIMARY KEY,
     market STRING NOT NULL,
     description STRING NOT NULL, 
-    creation_time TIMESTAMP NOT NULL, 
+    creation_time TIMESTAMP NOT NULL 
 );
 
 ALTER TABLE movr_pricing.promo_codes SET locality GLOBAL;
```
Comment (alexyarosh) [inline solutions/07-regional-tables/movr/pricing/data/pricing_database.sql:22]: This does not contain the solution to ex 07 (which involves setting locality of vip_rates)

File: setup/09-tradeoffs/movr/pricing/data/pricing_database.sql [hunk 7]
```diff
@@ -0,0 +1,26 @@
+CREATE DATABASE movr_pricing;
```
Comment (alexyarosh) [inline setup/09-tradeoffs/movr/pricing/data/pricing_database.sql:1]: init_db does not run this code

File: setup/09-tradeoffs/movr/pricing/data/pricing_database.sql [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    rate_code STRING PRIMARY KEY,
+    market STRING NOT NULL,
+    description STRING NOT NULL, 
+    creation_time TIMESTAMP NOT NULL 
+);
+
+ALTER TABLE movr_pricing.promo_codes SET locality GLOBAL;
+ALTER TABLE movr_pricing.vip_rates SET LOCALITY REGIONAL BY TABLE IN "us-west";
+
+ALTER DATABASE movr_pricing PLACEMENT RESTRICTED;
```
Comment (alexyarosh) [inline setup/09-tradeoffs/movr/pricing/data/pricing_database.sql:26]: FYI currently this is part of the exercise instructions (as opposed to part of the setup)

**Review by alexyarosh — CHANGES_REQUESTED** [review]

**Comment by alexyarosh** [comment]
@kikiya Overall a huger improvement, I think!
Some inconsistencies spotted. The biggest one is that the pricing DB is still not created in exercises where it needs to be, because init_db does not run pricing_data.sql

File: setup/09-tradeoffs/movr/pricing/data/pricing_database.sql (hunk 8 above)
Comment (kikiya) [reply setup/09-tradeoffs/movr/pricing/data/pricing_database.sql:26]: The exercise for this one should ask students to change this from placement `restricted` to `default`
I'll check on that!

File: setup/09-tradeoffs/movr/pricing/data/pricing_database.sql (hunk 7 above)
Comment (kikiya) [reply setup/09-tradeoffs/movr/pricing/data/pricing_database.sql:1]: included

File: solutions/07-regional-tables/movr/pricing/data/pricing_database.sql (hunk 6 above)
Comment (kikiya) [reply solutions/07-regional-tables/movr/pricing/data/pricing_database.sql:22]: added solution

File: setup/07-regional-tables/movr/pricing/data/pricing_database.sql (hunk 5 above)
Comment (kikiya) [reply setup/07-regional-tables/movr/pricing/data/pricing_database.sql:1]: included in init file

File: setup/05-global-tables/movr/pricing/data/pricing_database.sql (hunk 3 above)
Comment (kikiya) [reply setup/05-global-tables/movr/pricing/data/pricing_database.sql:1]: included in init file

File: setup/03-implement-multi-region-db/movr/vehicles/data/vehicles_database.sql [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+	('c7092d4f-228d-4570-aa64-576574e7f2ee', 85, '{"color": "black", "purchase_information": {"manufacturer": "Scoot Life", "purchase_date": "2020-07-05 19:13:58", "serial_number": "16360"}, "type": "scooter", "wear": "mint"}');
+
+INSERT INTO movr_vehicles.location_history (id, vehicle_id, ts, longitude, latitude) VALUES
+	('0011521c-7c34-4e48-9146-8980f6202996', 'e92dbdff-ee44-47ae-9ce3-b9b4eb1f7ea6', '2020-04-29 19:21:53+00:00', (-74.03534), 40.58763),
+	('00188ad3-fc76-4446-ba5e-e7517979342d', 'a4eea24e-669f-4e5f-85fc-7e88b970d1ab', '2020-04-29 23:07:49+00:00', (-73.98715), 40.74666),
+	('001a5a39-4f23-4b59-b245-cad6ddcaa808', 'd511f956-83e8-48ed-aa3b-4e0dd285169f', '2020-04-29 17:37:22+00:00', (-74.20637), 40.72875),
+	('002cf0db-64cd-4c93-b728-004c5b5723ef', '541f7135-41b6-484b-b201-ddb20ed25d47', '2020-04-29 20:23:31+00:00', (-74.28585), 40.54546),
+	('003813dc-1f95-46b5-b4c8-efb0ba016a3f', 'c7092d4f-228d-4570-aa64-576574e7f2ee', '2020-04-30 03:23:09+00:00', (-74.48876), 40.66873);
+
+ALTER DATABASE movr_vehicles SET PRIMARY REGION "us-east";
```
[the remark this answers is not in the document]
Comment (kikiya) [reply setup/03-implement-multi-region-db/movr/vehicles/data/vehicles_database.sql:36]: removed these configs

File: setup/03-implement-multi-region-db/movr/users/data/users_database.sql [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+
+CREATE TABLE movr_users.users (
+	email STRING PRIMARY KEY,
+	city STRING NULL,
+	last_name STRING NOT NULL,
+	first_name STRING NOT NULL,
+	phone_numbers STRING[] NOT NULL
+);
+
+ALTER DATABASE movr_users SET PRIMARY REGION "us-east";
```
[the remark this answers is not in the document]
Comment (kikiya) [reply setup/03-implement-multi-region-db/movr/users/data/users_database.sql:11]: removed these configs

File: setup/03-implement-multi-region-db/movr/rides/data/rides_database.sql [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+);
+
+CREATE TABLE movr_rides.events (
+    id UUID PRIMARY KEY,
+    ts TIMESTAMP DEFAULT now() NOT NULL,
+    event_type STRING NOT NULL,
+    event_data JSON NOT NULL
+);
+
+ALTER DATABASE movr_rides SET PRIMARY REGION "us-east";
```
[the remark this answers is not in the document]
Comment (kikiya) [reply setup/03-implement-multi-region-db/movr/rides/data/rides_database.sql:18]: removed these configs

**Review by alexyarosh — APPROVED** [review]
👍🏼

**Pull request merged**